awsagentprovider/src/main/java/com/softwareaws/xray/opentelemetry/providers/AwsAgentProperties.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.softwareaws.xray.opentelemetry.providers;
```

Comment:
jw, why are we using `com.softwareaws` instead of `software.amazon` or `com.amazonaws` as our namespace?